### PR TITLE
Completed HTML / responsive CSS / JS for Issue #1

### PR DESCRIPTION
**Pull request by josh-everett01**
I did have an issue with webpack and my JavaScript not linking with my HTML, but I went ahead and manually added my JavaScript file (which is stated not to do). I hope to get this cleared up soon.

### CODE REVIEW

File: src/css/style_two.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
+
```
Comment (Mycobee) [inline src/css/style_two.css:1]: can we rename and re-organize this css?  I feel maybe we should put all our login css in a file called login.css, and we can continue this pattern as we go (agent_dashboard.css, traveler_dashboard.css, new_trip.css), etc

File: src/index.js [hunk 2]
```diff
@@ -2,9 +2,32 @@
 // Do not delete or rename this file ********
 
 // An example of how you tell webpack to use a CSS (SCSS) file
-import './css/base.scss';
+// import './src/css/base.scss';
+// import './css'
```
Comment (Mycobee) [inline src/index.js:7]: I think this file should pretty much just require all the javascript files we need. 

inside that directory we can have another directory for our "helpers"
move this function in to a file called:
`src/javascripts/helpers/login_helper.js`

and require that file in `src/index.js`

We are going to be writing a lot of code, its best to keep it tidy early

File: src/index.js [hunk 3]
```diff
@@ -2,9 +2,32 @@
...
+    var h = document.createElement("H1"); // Create the H1 element 
+    var t = document.createTextNode("AGENT"); // Create a text element 
+    h.appendChild(t); // Append the text node to the H1 element 
+
+    document.body.appendChild(h); // Append the H1 element to the document body 
+  } else {
+    alert("Incorrect Login!!");
+    document.getElementById("input-username").value = "";
+    document.getElementById("input-password").value = "";
+
```
Comment (Mycobee) [inline src/index.js:29]: i do not dig this blank line

File: src/index.js [hunk 4]
```diff
@@ -2,9 +2,32 @@
...
+    document.body.appendChild(h); // Append the H1 element to the document body 
+  } else {
+    alert("Incorrect Login!!");
+    document.getElementById("input-username").value = "";
+    document.getElementById("input-password").value = "";
+    
+  }
+
+
```
Comment (Mycobee) [inline src/index.js:32]: or these

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,56 @@
...
+          </div>
+          <div id="password">
+            <label id="label-password" for="password">Password:</label><br />
+            <input
+              id="input-password"
+              type="text"
+              placeholder="Please enter your password"
+            /><br />
+          </div>
+          <button type="button" onclick="authPW()" id="submit">SUBMIT</button>
```
Comment (Mycobee) [inline src/index.html:46]: I'd like for us to figure out how to add this event listener somewhere else beside the html element.  essentially lets get an onclick in a javascript file that selects that button.  We don't need to call JS in our html files.  It works, but lets clean her up :)

File: src/css/style_two.css (hunk 1 above)
Comment (josh-everett01) [reply src/css/style_two.css:1]: Removed the CSS file I wasn't using and renamed the relevant CSS login.css.

**Comment by josh-everett01** [comment]
Will do, thanks!

**Comment by Mycobee** [comment]
Closes: #2

**Pull request merged**